src/content/docs/arctrl/datamap.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: "Datamap"
+lastUpdated: 2025-07-08
+authors:
+  - lukas-weil
+sidebar:
+  order: 4
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import { Card } from '@astrojs/starlight/components';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+The Datamap files used to annotate data files are modelled in ARCtrl as `Datamap` objects.
+
+They can be found in `Study`, `Assay`, `Workflow` and `Run` folders and therefore in the `ArcStudy`, `ArcAssay`, `ArcWorkflow` and `ArcRun` classes.
+
+![](@images/arctrl/datamap.png)
+
+The `Datamap` itself is a table with fixed columns. Each row in the Datamap represents and annotates a Data Fragment of a data file and is modelles as an `DataContext` object. The `Datamap` therefore consists of a collection of `DataContext` objects.
+
+In this small tutorial we will show how to create a `Datamap` and integrate it into an `ARC`. The `Datamap` will describe a CSV file called `sugar_result.csv` located in `assays/MyAssay/dataset`, with two columns, each containing measurements of cellular sugar content:
+
+<Card icon='seti:csv' title="sugar_result.csv">
+| sug_s1_r1 | sug_s1_r2 |
+|-----------|-----------|
+| 15.5      | 14.8      |
+| 15.2      | 15.1      |
+| 15.9      | 14.7      |
+</Card>
+
+
+## Preparation
+
+First, we will prepare some ontology terms that we will use in the `Datamap`.
+
+<Tabs syncKey="pl">
```

Comment:
By the way, these syncKeys can be used across sites on the KB, So if someone selects python here, they'd automatically get python pre-selected on other guides as well. 
So we might want to give them a more intuitive name.

Reply:
Hmm, pl = programming language.

Should we use the full name?

Reply:
although, it'll also not hurt to have `programmingLanguage` and `operatingSystem` (instead of OS)